docs/DeveloperGuide.md

```diff
@@ -50,7 +50,7 @@ The bulk of the app's work is done by the following four components:
 
 **How the architecture components interact with each other**
 
-The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `addp n/David p/98987676 e/david@example.com a/NUS t/student`.
```

Comment:
evrything else lgtm but it might be better to show examples for event addition as well.

Reply:
It will be pretty much the same as addp. I think one example might be enough.